code/game/gamemodes/blob/blobs/core.dm

```diff
@@ -14,6 +14,9 @@
 	processing_objects.Add(src)
 	if(!overmind)
 		create_overmind(new_overmind)
+		color = overmind.blob_reagent_datum.color

```

Comment:
I specifically removed this line as it causes runtimes, the line isn't needed as create_overmind() handles the colour now too. so please revert this back so it's the else chain